helm_deploy/offender-search/templates/_envs.tpl

```diff
@@ -8,7 +8,10 @@ env:
     value: "{{ .Values.env.JAVA_OPTS }}"
 
   - name: SPRING_PROFILES_ACTIVE
-    value: "elasticsearch"
+    value: "elasticsearch,stdout"
```

Comment:
I guess you can remove the stdout now? Or do it after appinsights issue is resolved?